content/blog/2017/2017-02-06-declarative-pipeline-ga.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax 1.0 is GA"
+tags:
+- pipeline
+- blueocean
+author: hrmpw
+---
+[NOTE]
+====
+This is a guest post by
+link:https://github.com/HRMPW[Patrick Wolf],
+Director of Product Management at
+link:http://cloudbees.com[CloudBees]
+and contributor to
+the Jenkins project.
+====
+
+I am very excited to announce the addition of
+link:https://plugins.jenkins.io/pipeline-model-definition[Declarative Pipeline syntax]
+1.0 to
+link:https://plugins.jenkins.io/workflow-aggregator[Jenkins Pipeline.]
+We think this new syntax will enable everyone involved in DevOps, regardless of expertise,
+to participate in the continuous delivery process. Whether creating, editing or reviewing
+a pipeline, having a straightforward structure helps to understand and predict the
+flow of the pipeline and provides a common foundation across all pipelines.
+
+= Pipeline as Code
+Pipeline as Code was one of the pillars of the Jenkins 2.0 release and an essential part of implementing continuous delivery. Defining all of the stages of an application's CD pipeline within a "Jenkinsfile" and treating it as part of the application code automatically provides all of the benefits inherent in SCM:
```

Comment:
" and treating it as part"
I would say it's not "treating".  Maybe "and having it checked-in as part"